core/pythonAction/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Dockerfile for python actions, overrides and extends ActionRunner from actionProxy
-FROM dockerskeleton
+FROM python:3-alpine
```

Comment:
this should remain as dockerskeleton (it allows us to have one base image for several different runtimes which we can pre-warm for example); can we just change the image in actionProxy instead?